saleor/checkout/calculations.py

```diff
@@ -257,23 +241,32 @@ def fetch_checkout_prices_if_expired(
     Prices can be updated only if force_update == True, or if time elapsed from the
     last price update is greater than settings.CHECKOUT_PRICES_TTL.
     """
-    checkout = checkout_info.checkout
+    from .utils import recalculate_checkout_discount
 
-    if not force_update and checkout.price_expiration < timezone.now():
+    checkout = checkout_info.checkout
+    if (
+        not force_update
+        and checkout.price_expiration
```

Comment:
when do we have a case that this value will be None?

Reply:
I need to make a final decision on it. I thought using `None` instead of `timezone.now()` might be easier for forced invalidation (and especially testing it) but I think I might go back to `timezone.now()`